packages/teams-js/test/private/copilot.spec.ts

```diff
@@ -384,4 +384,22 @@ describe('copilot', () => {
       });
     });
   });
+
+  describe('copilot.customTelemetry', () => {
+    describe('sendCustomTelemetryData', () => {
+      it('sendCustomTelemetryData should throw if called before initialization', async () => {
+        expect.assertions(1);
+        utils.uninitializeRuntimeConfig();
+        await expect(copilot.customTelemetry.sendCustomTelemetryData()).rejects.toThrowError(
+          new Error(errorLibraryNotInitialized),
+        );
+      });
+      it('sendCustomTelemetryData should throw if called before initialization', async () => {
```

Comment:
Copy/paste error? This test case is the successful case, right?

Reply:
resolved